### ISSUE TITLE
Check code policy before release of OTOBO 10.1.1

### ISSUE THREAD

**Issue by bschmalhofer** [RotherOSS/otobo#1629]
Looks like some glitches sneaked in after #1567.

**Comment by svenoe** [comment]
I got too many files with errors to correct this today. A significant fraction of them also seems to be due to the new "ProhibitUnusedVariables", which you recently included. I would rather not start this directly before we release, so we will postpone this.

**Comment by bschmalhofer** [comment]
Closing. No need to generally comply with the latest version of the Codepolicy for patch releases. Only the new code of the patch releases has to comply.
